app/src/main/java/com/example/bigowlapp/service/AlarmBroadcastReceiver.java

```diff
@@ -0,0 +1,87 @@
+package com.example.bigowlapp.service;
+
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
+import android.os.PowerManager;
+
+import androidx.core.app.NotificationCompat;
+import androidx.core.app.NotificationManagerCompat;
+
+import com.example.bigowlapp.R;
+import com.example.bigowlapp.model.Schedule;
+import com.google.firebase.firestore.GeoPoint;
+
+import static com.example.bigowlapp.utils.IntentConstants.EXTRA_LATITUDE;
+import static com.example.bigowlapp.utils.IntentConstants.EXTRA_LONGITUDE;
+import static com.example.bigowlapp.utils.IntentConstants.EXTRA_UID;
+
+/**
+ * The purpose of this BroadcastReceiver is to execute code after the time activation of an alarm.
+ * The alarms are set/defined in
+ * {@link com.example.bigowlapp.utils.AlarmBroadcastReceiverManager}
+ */
+public class AlarmBroadcastReceiver extends BroadcastReceiver {
```

Comment:
Might want to consider a name change to reflect the specific purpose of this alarm BroadcastReciever, especially in the case we need alarm receivers for other things. Name it based on the purpose of this specific alarm BroadcastReciever.